infra/lib/ci-cluster.ts

```diff
@@ -1,58 +1,80 @@
 import { Construct } from "constructs";
-import { aws_eks as eks, aws_ec2 as ec2, aws_iam as iam } from "aws-cdk-lib";
+import { aws_eks as eks, aws_ec2 as ec2, Stack } from "aws-cdk-lib";
+import * as blueprints from "@aws-quickstart/eks-blueprints";
 import * as cdk8s from "cdk8s";
-import { policies as ALBPolicies } from "./policies/aws-load-balancer-controller-policy";
 import {
   ProwGitHubSecretsChart,
   ProwGitHubSecretsChartProps,
 } from "./charts/prow-secrets";
 import {
-  EXTERNAL_DNS_NAMESPACE,
   FLUX_NAMESPACE,
   PROW_JOB_NAMESPACE,
   PROW_NAMESPACE,
 } from "./test-ci-stack";
+import {
+  GlobalResources,
+  ImportHostedZoneProvider,
+} from "@aws-quickstart/eks-blueprints";
 
-export type CIClusterCompileTimeProps = ProwGitHubSecretsChartProps;
+export type CIClusterCompileTimeProps = ProwGitHubSecretsChartProps & {
+  hostedZoneId: string;
+};
```

Comment:
huh.. TIL...

So the `&` operator can be used against a variable of type `object` to "append" properties to that object?

Reply:
Yes it's a union of the types. Typescript has some crazy type definition helpers

Reply:
Oh `ProwGitHubSecretsChartProps` is also a `type`, though, not an `object`